py3status/modules/xscreensaver.py

```diff
@@ -8,12 +8,14 @@
 
 This script is useful for people who let Xscreensaver manage dpms
 settings. Xscreensaver has its own dpms variables separate from xset.
-Dpms can be savely turned of in xset as long as xscreensaver is running.
+Dpms can be safely turned off in xset as long as xscreensaver is running.
 Settings can be managed using "xscreensaver-demo".
 
 Configuration parameters:
-    - format_off: string to display when Xscreensaver is disabled
-    - format_on : string to display when Xscreensaver is enabled
+    format_off   : string to display when Xscreensaver is disabled
+    format_on    : string to display when Xscreensaver is enabled
```

Comment:
There cannot be spaces between the config name and the `:` you can have as many spaces after as you want.

We look for `    <name>:`